department-of-veterans-affairs/va.gov-cms
# VACMS-16882: Resolves Error: Call to a member function getValue() on null

## Description
Resolves a bug in addToNationalOutreachCalendar() which can cause a php error.

Relates to #16882

## Testing done
Tested using different user roles, and using new and existing Event nodes.

## QA steps
As a content admin
1. Visit /node/add/event
2. Fill out the required fields
3. In the right sidebar, under "Feature This Content", check the "Featured" checkbox
4. Save the node
   - [x] Validate that there are no errors

### Definition of Done


### Select Team for PR review
